src/cdk/config.bzl

```diff
@@ -18,13 +18,12 @@ CDK_ENTRYPOINTS = [
     "text-field",
     "tree",
     "testing",
+    "testing/protractor",
+    "testing/testbed",
 ]
 
-# List of all entry-point targets of the Angular Material package. Note that
-# we do not want to include "testing" here as it will be treated as a standalone
-# sub-package of the "ng_package".
-CDK_TARGETS = ["//src/cdk"] + \
-              ["//src/cdk/%s" % ep for ep in CDK_ENTRYPOINTS if not ep == "testing"]
+# List of all entry-point targets of the Angular Material package.
+CDK_TARGETS = ["//src/cdk"] + ["//src/cdk/%s" % ep for ep in CDK_ENTRYPOINTS]
```

Comment:
It looks like this used to exclude `testing` but now does not. This is intentional right?

Reply:
Yes, I added a message for it in the PR description. I made `cdk/testing/*` real entry-points. Previously it was treated as sub-`ng_package`, but that rather made this config more confusing and less consistent. So now it's more consistent I'd say 😄